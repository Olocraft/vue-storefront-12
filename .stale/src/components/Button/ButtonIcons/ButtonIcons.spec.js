import { shallowMount } from '@vue/test-utils';
import ButtonIcons from './ButtonIcons.vue';

let wrapper;

describe('ButtonIcons.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(ButtonIcons);
  });
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
