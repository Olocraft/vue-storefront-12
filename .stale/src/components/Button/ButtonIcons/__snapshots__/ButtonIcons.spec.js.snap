// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonIcons.vue render component correctly 1`] = `
<button class="inline-flex items-center justify-center rounded-md cursor-pointer font-body
  disabled:cursor-not-allowed outline-violet button-primary button-medium"></button>
`;
